Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs

```diff
@@ -38,6 +43,8 @@
 using Swashbuckle.AspNetCore.SwaggerGen;
 using AuthenticationSchemes = Jellyfin.Api.Constants.AuthenticationSchemes;
 
+[assembly: InternalsVisibleTo("Jellyfin.Api.Tests")]
```

Comment:
This is normally placed in the csproj

Reply:
moved

Reply:
Had to roll back - did exactly the same as Mediabrowser.common.csj and what was said online - but it refused to compile.

Have left the info in the csj - and re-added it back in the file so it will compile. Could you take a look and see what i'm missing?

Reply:
You have to add the import to the csproj as well ;)

Reply:
I must be missing something really simple. I cannot find a way of importing System.Runtime.CompilerServices.

Have also tried to follow this, but cannot get it to work.

https://stackoverflow.com/questions/42810705/visual-studio-2017-and-the-new-csproj-internalsvisibleto

Reply:
Looking through the code, it seems like we use AssemblyInfo.cs more than the csproj one, so just put it in there.